data/cg-teams.ts

```diff
@@ -514,8 +514,8 @@ export default class TeamGenerator {
 		if (move.category === 'Special' && hasPhysicalSetup) powerEstimate *= 0.7;
 
 		const abilityBonus = (
-			((ABILITY_MOVE_BONUSES[ability] || {})[move.id] || 1) *
-			((ABILITY_MOVE_TYPE_BONUSES[ability] || {})[moveType] || 1)
+			((ABILITY_MOVE_BONUSES[toID(ability)] || {})[move.id] || 1) *
+			((ABILITY_MOVE_TYPE_BONUSES[toID(ability)] || {})[moveType] || 1)
```

Comment:
These previously weren't doing anything since `ability` is a name and `ABILITY_MOVE_BONUSES` is an ID table. I've changed it to do what was intended by the code, but we should probably be watchful for regressions.